src/easydmp/dmpt/admin.py

```diff
@@ -61,6 +61,28 @@ def get_canned_answers_for_user(user):
     return CannedAnswer.objects.filter(question__in=questions)
 
 
+class TemplateAuthMixin:
+    message_cannot_delete = ('The {} cannot be deleted because it is'
+                             ' in use by one or more plans.')
+
+    def has_change_permission(self, request, obj=None):
```

Comment:
We had this earlier I think: bad with mixin if it assumes something like this about super(). Better to inline or delegate. But your call.

Reply:
I'm thinking of making a class for this, containing this mixin and ObjectPermissionModelAdmin. "TemplatePermissionModelAdmin" maybe. Another time.